simulation/src/test/resources/report.md

```diff
@@ -4,7 +4,7 @@
                 all_nodes_500[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=50.0%	client_mean=PT0.6S         	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1000, 500=1000}
                     all_nodes_500[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=50.0%	client_mean=PT0.6S         	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1000, 500=1000}
                               all_nodes_500[UNLIMITED_ROUND_ROBIN].txt:	success=50.0%	client_mean=PT0.6S         	server_cpu=PT20M          	client_received=2000/2000	server_resps=2000	codes={200=1000, 500=1000}
-                   black_hole[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=30.0%	client_mean=PT0.6S         	server_cpu=PT6M           	client_received=600/2000	server_resps=600	codes={200=600}
+                   black_hole[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=89.0%	client_mean=PT0.600224719S 	server_cpu=PT17M48S       	client_received=1780/2000	server_resps=1780	codes={200=1780}
```

Comment:
@bwaldrep unsurprisingly this bugfix makes pin until error behave a lot better in the black hole case (https://github.com/palantir/dialogue/issues/615). We start off pinned to the bad host and chuck 20 requests into the black hole, at which point the concurrency limiter says no more, so we switch over to the non-broken host and everything after that succeeds.

Reply:
If we implement the feature to broadcast preferred node selection from a server, this algorithm could be helpful for services which benefit from cache locality but don't suffer correctness issues when requests don't hit the same host.